example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
     Built on chainkey cryptography, secure and true,
     An app that transcends, all we once knew...
 
-Principals: [] # TODO(NNS1-2293): This field will be useful in the future.
+# This is currently a placeholder field and must be left empty for now.
```

Comment:
what happens if it is not left empty? Will the validator catch this? 
If yes, I would leave as is.
If no, I would add a more prominent WARNING or so

Reply:
Added to NNS1-2508